src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -118,7 +118,7 @@ const WorkflowDashboard = _.flow(
     }), simplifiedFailures)
   }
 
-  const callNames = _.sortBy(_.identity, _.keys(calls))
+  const callNames = calls ? _.sortBy(callName => _.minBy(_.start, calls[callName]).start, _.keys(calls)) : []
```

Comment:
I can't find any documentation of `_.start`, did you mean `'start'`?

Reply:
(also, `_.keys` and `_.sortBy` should be already guarded to work normally if calls is undefined)

Reply:
That's because it's the magic "I don't want to specify a variable name" `_` rather than the lodash `_`

Reply:
... but since it's confusingly mixed in with all the lodashes, I guess I really ought to expand to give it a full function definition with a named input

Reply:
Does the `calls ? ...` at the start cover the guarding you're asking for?

Reply:
Right, I mean they should already be guarded, you shouldn't need the ternary at all. I think you should be able to use the string as a lodash shorthand, then you won't have to spin out a whole function.

Reply:
Oh wow, even nicer!